### ISSUE TITLE
Style/BracesAroundHashParameters autocorrect is invalid ruby

### ISSUE THREAD

**Issue by dduugg** [rubocop-hq/rubocop#7330]
## Expected behavior
given:
```ruby
# frozen_string_literal: true

def foo(sym, hash)
  puts sym, hash
end

foo :bar, {
  key: 'val',
}
```
where `.rubocop.yml` is
```
Style/TrailingCommaInHashLiteral:
  EnforcedStyleForMultiline: comma
```
then `bundle exec rubocop -a` should result in:
```ruby
# frozen_string_literal: true

def foo(sym, hash)
  puts sym, hash
end

foo :bar, 
  key: 'val'
```
(note the lack of a trailing comma in the invocation of `foo`)

## Actual behavior

Autocorrection results in:
```ruby
# frozen_string_literal: true

def foo(sym, hash)
  puts sym, hash
end

foo :bar, 
  key: 'val',
```
which is not valid ruby (trailing commas are not permitted in method invocations).

## Steps to reproduce the problem

☝️ 

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ bundle exec rubocop -V
0.74.0 (using Parser 2.6.4.0, running on ruby 2.4.5 x86_64-darwin17)
```

**Comment by pocke** [comment]
Thank you for reporting. But `Style/BracesAroundHashParameters` cop will be removed in the next release. https://github.com/rubocop-hq/rubocop/pull/7643

So I'll close this issue. :pray:
